services/director-v2/tests/unit/with_dbs/test_route_clusters.py

```diff
@@ -108,19 +109,21 @@ def creator(**overrides) -> Cluster:
         )
 
 
-def test_get_default_cluster_entrypoint(clusters_config: None, client: TestClient):
+async def test_get_default_cluster_entrypoint(
+    loop: AbstractEventLoop, clusters_config: None, async_client: httpx.AsyncClient
+):
     # This test checks that the default cluster is accessible
     # the default cluster is the osparc internal cluster available through a dask-scheduler
-    response = client.get("/v2/clusters/default")
+    response = await async_client.get("/v2/clusters/default")
```

Comment:
Why did you switch to an asyn client i.e. what different does it make to the test?

Reply:
I've already put a comment in a fixture but basically Starlette's TestClient creates a loop in a different thread, causing issues. Using the async client all code is attached to the same loop.